### PR TITLE
frozenset() initial addition

### LINKED ISSUE #4615: [Term Entry] Python sets .isfrozenset()
### Reason/inspiration (optional)

We would like a new term entry on the '.isfrozenset()' method for Sets in Python. The entry should go in a new file under `docs/content/python/concepts/sets/terms/isfrozenset/isfrozenset.md`

The entry should include the following:

- A description of the function
- A ## Syntax section that includes the syntax of the function and its details
- An ## Example section that includes a ```python ``` codeblock and ```shell ``` that shows the example output.
- A ## Codebyte Example that includes a ```codebyte/python ``` runnable codeblock

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by alimalim77**
### Description

Added the Term Entry for Python Concept `frozenset()` with details, description, syntax and code. 

### Issue Solved

- Created an entry for frozen set
- Added the required technical samples and details
- Closes #4615
### Type of Change

- Adding a new entry for frozenset()

### Checklist

- [ ✔️] All writings are my own.
- [✔️ ] My entry follows the Codecademy Docs style guide.
- [ ✔️] My changes generate no new warnings.
- [ ✔️] I have performed a self-review of my own writing and code.
- [ ✔️] I have checked my entry and corrected any misspellings.
- [ ✔️] I have made corresponding changes to the documentation if needed.
- [ ✔️] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ✔️] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ✔️] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/python/concepts/sets/terms/isfrozenset/isfrozenset.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+frozenset(iterable)
+```
+
+- `frozenset`: The `frozenset` created to store the unique values from `iterable`.
+- `iterable`: The container having the values to be stord in `frozenset`.
+
+## Example 
+
+```py 
+fs = frozenset([1, 2, 3, 4])
```
Comment (cigar-galaxy82) [inline content/python/concepts/sets/terms/isfrozenset/isfrozenset.md:33]:
```suggestion
fs1 = frozenset([1, 2, 3, 4])
```

File: content/python/concepts/sets/terms/isfrozenset/isfrozenset.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+fs2 = frozenset({5, 6, 7, 8})
+print(fs)
```
Comment (cigar-galaxy82) [inline content/python/concepts/sets/terms/isfrozenset/isfrozenset.md:35]:
```suggestion
print(fs1)
```

**Comment by cigar-galaxy82** [comment]
Hey @alimalim77

Thank you for contributing to Codecademy Docs! The entry is well written!

I've suggested a few changes modify them at your convenience

Thank you!

**Comment by alimalim77** [comment]
I have made the changes from my end. LNK if we are done with the changes or not. Thanks!

**Review by Sriparno08 — APPROVED** [review]
Looks good, @alimalim77! 👍

**Pull request merged**